write-fonts/src/tables/layout.rs

```diff
@@ -706,6 +706,23 @@ mod tests {
             .collect();
 
         assert!(builder.prefer_format_1());
+
+        fn iter_class_items(
```

Comment:
This seems like lots of cases in one test. Why isn't this three (one per let builder = ) different tests?

Reply:
no clear advantage to me either way, if you have a preference then happy to change it

Reply:
As a general pattern I've found more granular tests easier to work with than single tests that encompass many cases. It's very frustrating when you fix the test only to find out you fixed case 1 and that one test is actually 7 unit tests in one.

Reply:
okay I like that rationale.